docs/3x/PT_MixPrecision.md

```diff
@@ -0,0 +1,99 @@
+PyTorch Mixed Precision
+========================================
+
+1. [Introduction](#introduction)
+2. [Mixed Precision Support Matrix](#mixed-precision-support-matrix)
+3. [Get Started](#get-start)
+4. [Examples](#examples)
+
+## Introduction
+
+The recent growth of Deep Learning has driven the development of more complex models that require significantly more compute and memory capabilities. Several low precision numeric formats have been proposed to address the problem. Google's [bfloat16](https://cloud.google.com/tpu/docs/bfloat16) and the [FP16: IEEE](https://en.wikipedia.org/wiki/Half-precision_floating-point_format) half-precision format are two of the most widely used sixteen bit formats. [Mixed precision](https://arxiv.org/abs/1710.03740) training and inference using low precision formats have been developed to reduce compute and bandwidth requirements.
+
+The recently launched 3rd Gen Intel® Xeon® Scalable processor (codenamed Cooper Lake), featuring Intel® Deep Learning Boost, is the first general-purpose x86 CPU to support the bfloat16 format. Specifically, three new bfloat16 instructions are added as a part of the AVX512_BF16 extension within Intel Deep Learning Boost: VCVTNE2PS2BF16, VCVTNEPS2BF16, and VDPBF16PS. The first two instructions allow converting to and from bfloat16 data type, while the last one performs a dot product of bfloat16 pairs. Further details can be found in the [hardware numerics document](https://software.intel.com/content/www/us/en/develop/download/bfloat16-hardware-numerics-definition.html) published by Intel.  
+
+<p align="center" width="100%">
+    <img src="../imgs/data_format.png" alt="Architecture" height=230>
+</p>
+
+## Mixed Precision Support Matrix
+
+<table class="center">
+    <thead>
+        <tr>
+            <th>Framework</th>
+            <th>Backend</th>
+            <th>Backend Library</th>
+            <th>Backend Value</th>
+            <th>Support Device(cpu as default)</th> 
+            <th>Support BF16</th>
+            <th>Support FP16</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="1" align="left">PyTorch</td>
+            <td align="left">FX</td>
```

Comment:
Here mix-precision only covers fp16+fp32 or including advanced mix-precision like int8 + fp16+ fp32 ?  for latter case,  backend should be TorchDynamo in PT2e path, right ?

Reply:
The int8 is not supported here. Only a mixture of half-precision(bf16, fp16) and fp32 is available with this API.